config/migrations/2023/20230719103940-import-agb-addresses.sparql

```diff
@@ -0,0 +1,5451 @@
+
+      INSERT {
+        GRAPH ?g {
+          ?addressUri 
+            <https://data.vlaanderen.be/ns/adres#Adresvoorstelling.huisnummer> "9";
+            
+            <http://www.w3.org/ns/locn#postCode> "9300";
+            <http://www.w3.org/ns/locn#adminUnitL2> "OOST-VLAANDEREN";
+            <http://www.w3.org/ns/locn#thoroughfare> "Werf";
+            <https://data.vlaanderen.be/ns/adres#gemeentenaam> "Aalst";
+            <https://data.vlaanderen.be/ns/adres#land> "België";
+            <http://www.w3.org/ns/locn#fullAddress> "Werf 9, 9300 Aalst, België".
+        }
+      }
+      WHERE {
+        GRAPH ?g {
+          ?adminUnit <http://www.w3.org/2004/02/skos/core#prefLabel> "AGSA - Stadsontwikkeling Aalst".
+          ?adminUnit <http://www.w3.org/ns/org#hasSite> ?site.
```

Comment:
The prefixes here should be `hasPrimarySite`, not `hasSite` :grimacing: nothing gets inserted now from what I can see for most units